buildSrc/src/main/groovy/LessCompiler.groovy

```diff
@@ -1,10 +1,13 @@
-import org.gradle.api.tasks.*
+import org.gradle.api.tasks.InputDirectory
+import org.gradle.api.tasks.JavaExec
+import org.gradle.api.tasks.OutputFile
+import org.gradle.api.tasks.TaskAction
 
 class LessCompiler extends JavaExec {
   @InputDirectory
   def inputDir
 
-  @Input
+  @OutputFile
```

Comment:
This seems like a strange change. Is this file suddenly no longer an input but an output?

Reply:
yes, that was a leftover from some debugging. reverted